source/lex.tex

```diff
@@ -1370,7 +1370,7 @@
 
 \pnum
 \enternote A source-file new-line in a raw string literal results in a new-line in the
-resulting execution \term{string-literal}. Assuming no
+resulting execution \grammarterm{string-literal}. Assuming no
```

Comment:
@zygoloid: Ah, so that's what that means. I thought you could somehow edit the change in flight, but obviously that doesn't make any sense. Never mind then, we'll wait for the OP to edit his branch.